content/blog/redfish-interoperability-profiles.md

```diff
@@ -0,0 +1,239 @@
+
+The answer is "*yes*"; Redfish services not implementing properties not labeled "required" are compliant.
+
+This blog post elaborates the above statement and provides use cases where it can be a problem. Then it introduces the <a href="https://www.dmtf.org/dsp/DSP0272" target="_blank"> Redfish Interoperability Profiles Specification</a> that has been created by the Distributed Management Task Force (<a href="https://dmtf.org" target="_blank">DMTF</a>) to address those use cases.
+
+The Redfish Interoperability Profiles specification constitutes another Redfish specificity that could be added to the list presented in <a href="https://developer.hpe.com/blog/why-is-redfish%C2%AE-different-from-other-rest-apis-part-1/" target="_blank">part 1</a> and <a href="https://developer.hpe.com/blog/why-is-redfish%C2%AE-different-from-other-rest-apis-part-2/" target="_blank">part 2</a> of the *Why is Redfish different from other REST APIs* blog posts.
+
+### Redfish services can omit defined properties
+
+The DSP0266 standard document states in its <a href="https://www.dmtf.org/sites/default/files/standards/documents/DSP0266_1.20.1.html#properties" target="_blank">Properties overview</a> paragraph:
+
+* *Required properties shall always be returned in a response.*
+
+![Figure 1: DMTF Properties overview paragraph from DSP0266](/img/fig1-dmtfpropertiesoverview.png "Figure 1: DMTF Properties overview paragraph from DSP0266")
+
+<figcaption>Figure 1: DMTF Properties overview paragraph from DSP0266</figcaption>
+
+This assertion suggests that some properties are not "required" in the implementation of the service. As an example, in the data model of the <a href="https://redfish.dmtf.org/schemas/v1/Manager.v1_19_1.json" target="_blank">Baseboard Manager Controller</a> (BMC) (Figure 2), the only required properties are: `@odata.id`, `@odata.type`, `Id` and `Name`. Since the `FirmwareVersion` property is not listed in this normative document, its implementation is not required in Redfish services.
+
+![Manager required properties](/img/fig2-managerrequiredproperties.png "Manager required properties")
+
+<figcaption>Figure 2: Manager required properties</figcaption>
+
+Requiring the implementation of only a small number properties *provide\[s] significant flexibility, and allow conforming implementations on a wide variety of products* as mentioned in the abstract of the <a href="https://www.dmtf.org/sites/default/files/standards/documents/DSP0272_1.8.0.pdf" target="_blank">standard document</a>
+
+### Who cares?
+
+Flexibility is great, but too much flexibility can be a problem for standard organizations, sofware projects or end users willing to move from the Intelligent Platform Management Interface (<a href="https://www.intel.com/content/www/us/en/products/docs/servers/ipmi/ipmi-home.html" target="_blank">IPMI</a>) to Redfish for hardware management.
```

Comment:
to move **away** from the Intelligent Platform Management Interface (<a href="https://www.intel.com/content/www/us/en/products/docs/servers/ipmi/ipmi-home.html" target="_blank">IPMI</a>) to Redfish for hardware management.